src/animationBuilder.tsx

```diff
@@ -0,0 +1,60 @@
+import type {
+  ILayoutAnimationBuilder,
+  LayoutAnimationFunction,
+} from './reanimated2/layoutReanimation';
+import type { StyleProps } from './reanimated2/commonTypes';
+
+const mockTargetValues = {
```

Comment:
Let's add to this proper type `LayoutAnimationsValues` to keep this mock up to date with type.